.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+
+jobs:
+  build:
+
```

Comment:
Matter of style, so feel free to ignore but IMO, we can drop empty lines after opening a "tag" (here with "build" and below with "steps"), and maybe also after "checkout"

Reply:
I tried to group them for what they do. I guess, we can go over it if the file grows larger in size.